services/webnn/ort/graph_builder_ort.h

```diff
@@ -94,17 +94,24 @@ class GraphBuilderOrt {
   const mojom::Operand& GetOperand(uint64_t operand_id);
   std::string GetOperandName(uint64_t operand_id);
 
-  // Some initializers must be uploaded to raw data, e.g. Reshape op needs
-  // parameter *shape* as raw data to do shape inference.
+  // Create a new initializer copied into graph.
+  uint64_t NewInitializer(base::span<const uint32_t> shape,
+                          base::span<const uint8_t> data,
+                          OperandDataType data_type,
+                          bool as_raw_data);
+  // Some initializers must be uploaded to raw data, e.g. Reshape and Expand op
+  // need parameter *shape* as raw data to do shape inference. Slice op needs
+  // parameter *starts*, *ends* and *steps* as raw data to do shape inference.
   //
   // IssueA: Are there other ops requiring initializer as raw data? Currently we
   // upload all constants into external data (for potential zero-copy) but what
   // if some ops like Reshape can't take external data as initializers?
```

Comment:
Is this issue resolved?

Reply:
> Is this issue resolved?

I think we might need to add more as we go along to find if there are any similar cases.

Reply:
Link to https://github.com/shiyi9801/chromium/issues/52